zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -355,8 +357,8 @@ where
     )
 }
 
-/// Select transaction inputs, compute fees, and construct a proposal for a transaction
-/// that can then be authorized and made ready for submission to the network with
+/// Select transaction inputs, compute fees, and construct a proposal for a transaction or series
+/// of transactions that can then be authorized and made ready for submission to the network with
 /// [`create_proposed_transaction`].
```

Comment:
```suggestion
/// [`create_proposed_transactions`].
```
(I know this is already merged.)

Reply:
Fixed in #1105